WtWSMS/common/culture/cultures/BP_east_slavic.txt

```diff
@@ -1,7 +1,12 @@
-﻿old_slavic = {
+﻿#Notes:
+#[1] - From what I've read of proto-Slavic groups, they were fairly settled and had, at most, a relatively
+# flat social structure, with social stratification occurring when population density increased. Collective
+# lands seems to be the "best fit" for this notion of a flat hierarchy
```

Comment:
According to Peter Heather in Empires and Barbarians, it was during the 6th century that most Slavs underwent socio-political changes leading to an increasing stratification. Prior to that there are indeed no attested mentions of Slavic leaders in for example Procopius. So I don't know if we should add mechanics for this to evolve or if we should at start keep the previous setup since it was underway.

Reply:
I have no strong feelings one way or the other. But it strikes me as more of a 0.2.0 for more extensive mechanic changes like that. I think "Collective Lands" works as a stop-gap for 0.1.3 though.

Reply:
Ok, fine by me to include this as 0.2.0 material, I would just add a comment to remember it given the general workload of that update 😉

Reply:
Commit this comment in the Design notes for 0.2.0